src/sentry/static/sentry/app/views/availableWork/index.jsx

```diff
@@ -0,0 +1,46 @@
+import ClimsTypes from 'app/climsTypes';
```

Comment:
I have a feeling that this view is replacing another view, which is now redundant. If you don't already have deleted this file, please do it now :)